server/services/DataStreamService.ts

```diff
@@ -63,24 +72,35 @@ export default class DataStreamService {
   };
 }
 
+export const CUSTOM_DATA_STREAM_SECURITY_EXCEPTION = "[data_stream_security_exception]";
+export const DATA_STREAM_LACK_PERMISSION_WARNING =
```

Comment:
nit: Only used on frontend, can add to frontend constants instead of keeping in server dir